src/config.js

```diff
@@ -1,3 +1,12 @@
+
+function normalizeAccessToken(config) {
+  // eslint-disable-next-line
+  console.warn('[ShopifyBuy] accessToken is deprecated as of v1.0, please use storefrontAccessToken instead.');
+  const accessToken = config.storefrontAccessToken || config.accessToken;
```

Comment:
ah, good point. the function is pretty tightly coupled with that block anyway, so i'll just make it inline.